app/services/food_service.rb

```diff
@@ -0,0 +1,20 @@
+# app/services/food_service.rb
+
+class FoodService 
+  def self.connection
+    Faraday.new(url: 'https://api.nal.usda.gov/fdc/v1/') do |faraday|
+      faraday.params['api_key'] = ENV['food_api_key']
+    end
+  end
+
+  def self.foods_containing_ingredient(query)
+    response = connection.get('foods/search') do |request|
+      request.params['api_key'] = ENV['food_api_key']
+      request.params['query'] = ENV["#{query}"]
```

Comment:
I don't think you're saving the query coming in from the controller as an environment variable anywhere, so we can use `query` on its own here:
`request.params['query'] = query`